firebase-firestore/CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Unreleased
+* [fixed] Implement equals method on Filter class. [#5210](//github.com/firebase/firebase-android-sdk/issues/5210)
+
+# 24.7.0
 * [feature] Expose MultiDb support in API. [#4015](//github.com/firebase/firebase-android-sdk/issues/4015)
 * [fixed] Fixed a thread interference issue that may lead to a ConcurrentModificationException.
-    (GitHub [#5091](//github.com/firebase/firebase-android-sdk/issues/5091){: .external})
-* [fixed] Implement equals method on Filter class. [#5210](//github.com/firebase/firebase-android-sdk/issues/5210)
+  (GitHub [#5091](//github.com/firebase/firebase-android-sdk/issues/5091){: .external})
```

Comment:
Is this change of indentation intentional?

Reply:
it was just a copy paste but in hind sight i think the current one is the correct identation